packages/gen-ai-hub/README.md

```diff
@@ -0,0 +1,32 @@
+# @sap-ai-sdk/gen-ai-hub
+
+This package incorporates generative AI into your AI activities in SAP AI Core and SAP AI Launchpad.
+
+### Installation
+
+```
+$ npm install @sap-ai-sdk/gen-ai-hub
+```
+
+## Pre-requisites
+
+- [Enable the AI Core service in BTP](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/initial-setup)
+- Project configured with Node.js v20 or higher and native ESM support enabled
+- For testing your application locally:
+  - Download a service key for your AI Core service instance
+  - Create a `.env` file in the sample-code directory
+  - Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`
+
+## Usage
+
+<!-- Details to be added -->
```

Comment:
Is there at ticket for this or is this WIP?

Reply:
There are 2 tickets in the input queue for adding orchestration and openai client docs